schema/PMD.yaml

```diff
@@ -0,0 +1,43 @@
+$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-ts/ts_pmd/blob/master/schema/PMDevice.yaml
+title: PMD v1
+description: Schema for PMD configuration files
+definitions:
+  hub_specific_schema:
+    type: object
+    properties:
+      sal_index:
+        type: number
```

Comment:
Please add `minValue: 1` (I may have the keyword wrong) and a description field.